packages/twenty-ui/src/input/button/components/Button.tsx

```diff
@@ -69,7 +69,6 @@ const StyledButton = styled('button', {
                   : theme.background.transparent.light
                 : theme.background.transparent.light};
               border-width: 1px 1px 1px 1px !important;
-              opacity: ${disabled ? 0.24 : 1};
```

Comment:
I guess this has been validated with design!
Agree opacity does not seem the right implementation but are we still OK with the new disabled behavior?